### DISCUSSION TITLE
remove autoexec. ipxe and change banner name

### DISCUSSION THREAD

**Discussion by tonho911** [ipxe/ipxe#1237] [Q&A]
does anyone know how to
how to remove autoexec. ipxe. .?
1 IPXE Initialising devices... autoexec. ipxe. .. Not found (https://Ipxe.org/2d12618e)

thanks

**Comment by v01t** [comment]
I could be wrong but I didn't found any way to disable it, so in my case I keep ipxe build's for BIOS and EFI clean (without any extra embed logic) and instead move it to `autoexec.ipxe`, also just having empty file resolve the issue

**Comment by mcb30** [comment]
> I could be wrong but I didn't found any way to disable it, so in my case I keep ipxe build's for BIOS and EFI clean (without any extra embed logic) and instead move it to `autoexec.ipxe`, also just having empty file resolve the issue

Yes, that's exactly the idea. For almost all use cases, users should be able to use autoexec.ipxe instead of having to recompile with an embedded script.

**Comment by tonho911** [comment]
@v01t  @mcb30 
How do I do it? For example, if I want to run autoexec.ipxe with a program, for example gamemenu.exe, sir, so that when Windows boots, the program will run automatically without me messing with shell:startup?

Please teach me, sir, maybe this can help others, thanks

**Reply by mcb30** [reply]
`autoexec.ipxe` controls what iPXE does.  It has nothing to do with what Windows does after Windows has started up.
